### PR TITLE
fixed incomplete url

### PR DESCRIPTION
**Pull request by kai3x5**
rebuilt and tested locally.

### CODE REVIEW

**Comment by kalyanceg** [comment]
Thanks for the catch
Let me merge this

File: courses/databases_sql/intro.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Relational Databases
 
 ### Prerequisites
-*   Complete [Linux course](/linux_basics/intro/)
+*   Complete [Linux course](https://linkedin.github.io/school-of-sre/linux_basics/intro/)
```
Comment (sanketplus) [inline courses/databases_sql/intro.md:4]: we should be using relative links for internal documents. re: https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages

current one will need to change if we shifted to other domain. mkdocs supports linking internal docs via relative paths

Comment (kalyanceg) [reply courses/databases_sql/intro.md:4]: can you open an issue @sanketplus ?
We can audit and standardise to relative URLs

Comment (kai3x5) [reply courses/databases_sql/intro.md:4]: ironically this was the only link in the entire document using relative url. literally every other link was an absolute url.

Comment (sanketplus) [reply courses/databases_sql/intro.md:4]: no worries @kai3x5 might have gone unnoticed earlier. have opened #52

**Pull request merged**